src/student/components/quizView/QuestionsList.tsx

```diff
@@ -10,8 +10,25 @@ export function QuestionsList() {
     let answers = new Map();
     const [quiz, setQuiz] = useState(testData());
     const [quizID, setQuizID] = useState("");
+
+    const fillcheckboxValues = () => {
+        let values : boolean[][] = [];
+        for(let i = 0; i < quiz.questions.length; i++) {
+            values[i] = []
+            let len;
+            if(quiz.questions[i].options === undefined) continue;
+            else{  len = quiz.questions[i].options!.length;}
+            for(let j = 0; j < len; j++)
```

Comment:
Why this weird `len` variable instead of `j < quiz.questions[i].options!.length`?